### DISCUSSION TITLE
[Forge] Flux Realistic, further details please

### DISCUSSION THREAD

**Discussion by aggregate15** [lllyasviel/stable-diffusion-webui-forge#1732] [Q&A]
I have used the new sampler **[Forge] Flux Realistic** but other than slower speeds, I am not seeing much of a difference when it comes to realism ? In fact the images with this new sampler seem like they are generated at higher CFG (more plastic looking skin than other samplers). 

Below image for comparison, the one on the right is using the new sampler.  The skin texture and surroundings are much worse on the right, The only thing is that in the left image, the plate is being held up just by the knuckles, which is not the case in the Flux Realistic image (maybe that counts to realistic physics at the least, or it may just a random chance). But overall Flux Realistic looks worse in terms of realism, that too, with a 70% slower  generation time. Can someone please explain what's specific about this sampler and how to use it properly for realism ? This is not to diss on the amazing new features being provided by Forge. Its Great to have another sampler, but maybe I'm doing something wrong so as not to achieve the full potential of this sampler. 
 
![Picture2](https://github.com/user-attachments/assets/54a2ec1b-4fca-4406-be85-218683747b17)

**Comment by pflky** [comment]
Set it down to around 2 CFG, 3.5 CFG is only really if you really want it to follow prompts more, especially when it comes to text. Most people forget that CFG is sort of a way to get more adherence without having to increase steps. I also tend to lean toward DDIM or Beta as the scheduler in Flux. 

![00013-1114416060- Forge  Flux Realistic-20-1-flux1-dev-Q8_0](https://github.com/user-attachments/assets/fe06cf98-e9ac-46c8-b515-1db73a324b13)

Notice how the contrast isn't as high in 2 CFG images, they look far more natural. You will of course run into potential adherence issues at 2 CFG, but when you do get a keeper it's generally better than any keeper from 3.5 CFG. This image makes your left picture look plastic in comparison. Another thing to note is when you decrease CFG, you also generally increase steps. It's not an all the time rule, but 40-60 steps at 2 CFG can deliver very good images, and at 2 CFG, more steps can generally lead to more adherence, so steps essentially become the adherence slider instead. The trick is to balance CFG with steps.

**Reply by GPU-server** [reply]
What's "beta"? I don't see any sampler of that name.

**Reply by maraan666** [reply]
It's not a sampler, it's a schedule type (the  next box!). It works well with Euler and Flux Realistic, arguably better than "simple".

**Comment by GPU-server** [comment]
Since I have you here, what do you think are the best samplers for FLUX guys?
Which one to use for what?

**Reply by maraan666** [reply]
I like Flux Realistic with the Beta schedule for pretty much everything with Flux.

**Reply by GPU-server** [reply]
> I like Flux Realistic with the Beta schedule for pretty much everything with Flux.

So you dont make much anime or "style" images? I would also would like to know what are the best for this option.

**Reply by maraan666** [reply]
Actually, Flux Realistic works excellently for images that aren't "realistic". Try it. There is no such thing as a "best" sampler, it's a matter of personal artistic preference.

**Reply by aggregate15** [reply]
I tried all of them and IPNDM (Simple) gives me best results for 'realistic looking' images with same speed as Euler, etc. I have tried a lot of other options but I have found al of them to be slower or giving that fake/plastic look or blurred backgrounds.

**Reply by bearbearprints** [reply]
@aggregate15 - when I use IPNDM - simple, I get weird "low quality" look. Texture and realism good, but I think IPNDM doesn't like something in my settings.

**Comment by krajacic** [comment]
Is there a way to download and use Forge Realistic in Comfyui? Thanks

**Reply by MrSiggwell** [reply]
Have you found it yet? im still looking.

**Reply by krajacic** [reply]
You'll need to downgrade your Forge to have it :(
This is the only way.

**Comment by danilomaiaweb** [comment]
For comparison, I created these images...
20 steps, Schedule type Simple, 1024x1024
I had to crop the images because they were too big to post here.

Please see:
![0101](https://github.com/user-attachments/assets/8f01c96e-6b9b-4caf-8b6d-75768f966f6a)
![0202](https://github.com/user-attachments/assets/28391f33-5134-493b-b4d7-5cb9ed85c050)
![0303](https://github.com/user-attachments/assets/35da2e89-2c28-4a8c-a055-069b1502a501)
![0404](https://github.com/user-attachments/assets/4a7ea042-5bb5-42d5-9962-99e6aa4b3de5)

**Reply by BEBOP0093** [reply]
Hello! I am new to FLUX. I would like to know what settings you use to obtain those realistic photos. I can't get them to turn out so perfect. Thanks in advance

**Reply by ScionoicS** [reply]
The sampler code is gone and you can't use it now.  Thank Comfy Org for flipping out about GPL3 licensing like they've never copied any code themselves.

**Reply by Leee1414** [reply]
So now that the [Forge] Flux Realistic sampler has been taken from us  (it was my "go-to" sampler), any suggestions on the next best sampler to use?

**Reply by dilectiogames** [reply]
I'm afraid to update forge, can I save the samplers so they don't get deleted? where are located and how to save them and use them later?

**Reply by Examoontv** [reply]
> The sampler code is gone and you can't use it now. Thank Comfy Org for flipping out about GPL3 licensing like they've never copied any code themselves.

it wasnt comfy who flipped out it was some concern troll with a profile named after a bridge troll from an old game series called touhou.  The requester has no real github post history yet his request got accepted by some jannie sabotaging the project.  This behavior is why most open source projects dont last very long. Sooner or later some clown will sabotage the project.

**Reply by HubertPoros** [reply]
Hello,

Can someone please package the application versions from [Forge] Flux Realistic and make them available for download ? I would be very grateful

**Reply by sanhushuk321-bit** [reply]
Hello! could you tell me where I can download the Flux Realistic sampler for Forge? I don't have it in the sampler list.
